packages/ketcher-core/src/application/render/restruct/rergroup.js

```diff
@@ -63,19 +63,34 @@ class ReRGroup extends ReObject {
 
     if (ret) ret = ret.extend(BORDER_EXT, BORDER_EXT)
 
+    let attachmentPointsVBox = render.ctab.getAttachmentsPointsVBox(
+      this.getAtoms(render)
+    )
+    if (attachmentPointsVBox)
+      attachmentPointsVBox = attachmentPointsVBox.extend(BORDER_EXT, BORDER_EXT)
+
+    ret =
```

Comment:
If you understood what `ret` means, I would like to ask you to rename it

Reply:
sure, will do